tailwind.config.js

```diff
@@ -1,11 +1,44 @@
 module.exports = {
-  purge: [],
+  purge: ["src/**/*.{ts,tsx}"],
   darkMode: false, // or 'media' or 'class'
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        violet: {
+          light: "#C4D1FE",
+          dark: "#3C3CBE",
+          DEFAULT: "#6465F1",
+        },
+      },
+    },
+    container: {
+      padding: {
+        DEFAULT: "1rem",
+        sm: "2rem",
+        lg: "4rem",
+        xl: "5rem",
+        "2xl": "6rem",
+      },
+      screens: {
```

Comment:
I wouldn't recommend changing the default breakpoints of Tailwind. You can use their default breakpoints like `sm:`, `lg:`, etc. without having to customize the `screens` here. You can remove the `screens` property.